editor.planx.uk/src/@planx/components/Pay/Public/Pay.tsx

```diff
@@ -268,6 +271,35 @@ function Component(props: Props) {
       ) : (
         <DelayedLoadingIndicator text={state.displayText || state.status} />
       )}
+      {/* session id exposed for testing purposes */}
```

Comment:
This can also be accessed via the store (exposed on `window` if that can be accessed via Playwright) or the URL which might be a more repeatable solution that potentially adding `data-sessionid` across multiple elements in future (I could see a lot of tests relying on it).